chain/client/src/client.rs

```diff
@@ -71,6 +73,18 @@ pub struct ClientActor {
     num_tx_processed: u64,
 }
 
+fn wait_until_genesis(genesis_time: &DateTime<Utc>) {
+   let now : DateTime<Utc> = Utc::now();
```

Comment:
don't need type here? usually if the types are not required, it's preferred to skip them (also Clion does show the inferred types)

Reply:
Yeah, it makes sense to remove these types